backend/src/main/java/com/benine/backend/database/Database.java

```diff
@@ -162,4 +160,55 @@
    * @param preset The preset
    */
   void deleteTagFromPreset(String tag, Preset preset);
+  
+  /**
+   * delete the tags from a preset.
+   * @param preset to delete the tags for.
+   */
+  void deleteTagsFromPreset(Preset preset);
+
+  /**
+   * Gets a list of presets belonging to the concert.
+   *
+   * @param queueID The ID of the concert to get the presets from
+   * @return The list of presets
+   */
+  ArrayList<Preset> getPresetsList(int queueID);
+
+  /**
+   * Adds a list of presets to an existing queue.
+   *
+   * @param presets The list which needs to be added
+   * @param queueID The id of the queue
+   */
+  void addPresetsList(ArrayList<Preset> presets, int queueID);
```

Comment:
I also added a (string) name so that it is easier to use in the UI. Otherwise you will only be able to see a number.